src/config.hpp.in

```diff
@@ -55,6 +55,9 @@
 // (enabled by default)
 #define @EXCEPTION_HANDLING_OPTION@
 
+// Use lightweight kernels
```

Comment:
**Not for this PR**, but we should consider moving the contents of this file to the parthenon namespace and converting as many of the macros to constexprs as possible.

Reply:
Created #241 to keep track of this.